repose-aggregator/core/core-lib/src/main/java/org/openrepose/powerfilter/PowerFilter.java

```diff
@@ -374,6 +382,10 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
                 }
                 requestFilterChain.startFilterChain(mutableHttpRequest, mutableHttpResponse);
             }
+        } catch (InvalidMethodException ime) {
+            LOG.debug("{}:{} -- Invalid HTTP method requested: {}", clusterId, nodeId, mutableHttpRequest.getMethod(), ime);
+            mutableHttpResponse.sendError(HttpServletResponse.SC_BAD_REQUEST, "Error processing request");
+            mutableHttpResponse.setLastException(ime);
```

Comment:
It works, so I :+1: 

However, I wonder which is cleaner/clearer: this approach, or handling the check in its own try/catch block earlier in the method?